.make-packages.js

```diff
@@ -1,136 +1,62 @@
-"use strict";
-
-let pkg = require('./package.json');
-let fs = require('fs-extra');
-let mkdirp = require('mkdirp');
-let path = require('path');
-let klawSync = require('klaw-sync');
-let licenseTool = require('./tools/add-license-to-file');
-let addLicenseToFile = licenseTool.addLicenseToFile;
-let addLicenseTextToFile = licenseTool.addLicenseTextToFile;
-let makePackages = require('./.make-helpers');
-let copySources = makePackages.copySources;
-let cleanSourceMapRoot = makePackages.cleanSourceMapRoot;
-let bo = null;
-// Build Optimizer is not available on Node 4.x. Using a try/catch
-// here to make sure the build passes on Travis using Node 4, but
-// the NPM distribution will run through build-optimizer.
-try {
-  bo = require('@angular-devkit/build-optimizer');
-} catch (e) {}
-
-
-const ROOT = 'dist/';
-const CJS_ROOT = ROOT + 'cjs/';
-const ESM5_ROOT = ROOT + 'esm5/';
-const ESM2015_ROOT = ROOT + 'esm2015/';
-const UMD_ROOT = ROOT + 'global/';
-const ESM5_FOR_ROLLUP_ROOT = ROOT + 'esm5_for_rollup/';
-const TYPE_ROOT = ROOT + 'typings/';
-const PKG_ROOT = ROOT + 'package/';
-const CJS_PKG = PKG_ROOT + '';
-const ESM5_PKG = PKG_ROOT + '_esm5/';
-const ESM2015_PKG = PKG_ROOT + '_esm2015/';
-const UMD_PKG = PKG_ROOT + 'bundles/';
-const SRC_ROOT_PKG = PKG_ROOT +  'src/';
-const TYPE_PKG = PKG_ROOT;
-
-// License info for minified files
-let licenseUrl = 'https://github.com/ReactiveX/RxJS/blob/master/LICENSE.txt';
-let license = 'Apache License 2.0 ' + licenseUrl;
-
-delete pkg.scripts;
-fs.removeSync(PKG_ROOT);
-
-let rootPackageJson = Object.assign({}, pkg, {
-  name: 'rxjs',
-  main: './index.js',
-  typings: './index.d.ts',
-  module: './_esm5/index.js',
-  es2015: './_esm2015/index.js'
+const { writeFileSync } = require('fs');
+const { cp, exec, ls, mkdir, rm } = require('shelljs');
+const { buildOptimizer } = require('@angular-devkit/build-optimizer');
+const pkg = require('./package.json');
+
+// TODO: 
+// - what about sourcemaps? 
+// Angular itself has inline sourcemaps. I don't see the point, since you don't have sources
+// on the package.
```

Comment:
I'm not sure what to do about sourcemaps. I think the current ones are a bit weird and work only because the TS sourcefiles are bundled. We could inline sourcemaps but I'm not sure if that impacts parsing performance.